examples/inverse/plot_use_sourcemorph.py

```diff
@@ -0,0 +1,250 @@
+                        reject=dict(grad=4000e-13, mag=4e-12, eog=150e-6))
+    evoked = epochs.average()
+
+    # Read regularized noise covariance and compute regularized data covariance
+    noise_cov = mne.compute_covariance(epochs, tmin=tmin, tmax=0,
+                                       method='shrunk')
+    data_cov = mne.compute_covariance(epochs, tmin=0.04, tmax=0.15,
+                                      method='shrunk')
+    # Read forward model
+    forward = mne.read_forward_solution(fname_fwd)
+
+    # Compute weights of free orientation (vector) beamformer with weight
+    # normalization (neural activity index, NAI). Providing a noise covariance
+    # matrix enables whitening of the data and forward solution. Source
+    # orientation is optimized by setting pick_ori to 'max-power'.
+    # weight_norm can also be set to 'unit-noise-gain'. Source orientation can
+    # also be 'normal' (but only when using a surface-based source space) or
+    # None, which computes a vector beamfomer. Note, however, that not all
+    # combinations of orientation selection and weight normalization are
+    # implemented yet.
+    filters = make_lcmv(evoked.info, forward, data_cov, reg=0.05,
+                        noise_cov=noise_cov, pick_ori='max-power',
+                        weight_norm='nai')
+
+    # Apply this spatial filter to the evoked data.
+    stc = apply_lcmv(evoked, filters, max_ori_out='signed')
+
+    # take absolute values for plotting
+    stc.data[:, :] = np.abs(stc.data)
+
+    # Save result in stc files if desired
+
+    # select time window (tmin, tmax) in ms - consider changing for real data
+    # scenario, since those values were chosen to optimize computation time
+    stc.crop(0.087, 0.087)
+
+    src = forward['src']
+
+    if fname is not None:
+        stc.save(fname)
+
+    return stc, src
+
+
+# Setup path
+data_path = mne.datasets.sample.data_path()
+t1_sample = data_path + '/subjects/sample/mri/brain.mgz'
+t1_fsaverage = data_path + '/subjects/fsaverage/mri/brain.mgz'
+fname_fwd = data_path + '/MEG/sample/sample_audvis-meg-vol-7-fwd.fif'
+subjects_dir = data_path + '/subjects'
+
+# read precomputed data
+compute_lcmv_example_data_surf(data_path, fname='lcmv-surf')
+compute_lcmv_example_data_vol(data_path, fname='lcmv-vol')
+
+src = mne.read_forward_solution(fname_fwd)['src']
+stc = mne.read_source_estimate('lcmv-vol-vl.stc')
+stc_rh = mne.read_source_estimate('lcmv-surf-rh.stc')
+
+###############################################################################
+# Morph VolSourceEstimate
+
+# initialize morpher for VolSourceEstimate (src=src)
+morpher = SourceMorph('sample',
+                      'fsaverage',
+                      src=src,
+                      subjects_dir=subjects_dir,
+                      grid_spacing=(5., 5., 5.))
+
+# save morph
+morpher.save('vol')
+
+# load morph by initializing with file name
+morpher_loaded = SourceMorph(None, None, fname='vol-morph.h5')
```

Comment:
you should have a read function. Like:

read_source_morph

to be consistent.

Reply:
also returned instance should be called

source_morph

to be consistent. Morpher does not match class name and _loaded is unnecessary.

Reply:
agree on both - but on the first with a sad eye as well, because I found it so intuitive - but agree: consistency